locals.pkr.hcl

```diff
@@ -31,6 +31,18 @@ locals {
   }
   # Must be greater than 127 Gb to allow Azure template for Windows
   disk_size_gb = 150
+
+  jdk_infos = yamldecode(file("jdks-infos.yaml"))
+  # Mapping architecture to temurin jdk defaults, see https://api.adoptium.net/q/swagger-ui/#/Types/getArchitectures
+  mapped_arch = lookup({
+    "amd64"  = "x64"
+    "arm64"  = "aarch64"
+  }, var.architecture, "x64") # fallback x64
```

Comment:
I would keep using the `amd64` and `arm64` values in the YAML. The fact thet Temurin utilizes other platform strings is not a concern to us since we only provide URLs.

Adding this mapping here adds complexity for no benefit (unless I missed something).